nservicebus/messaging/routing-extensibility_content_core_[,6).partial.md

```diff
@@ -0,0 +1,13 @@
+The routing system can be extended by means of providing a routing configuration from source different than standard (`app.config` file) either by using a configuration source:
```

Comment:
should that be sources?

would "from different sources than" read better?

> (`app.config` file)

why not mentioning the code first API?

How about:

> The routing system can be extended by providing custom sources of routing information to extend or replace the standard routing configuration APIs.